### ISSUE TITLE
Combined output from `signac find` and `signac statepoint`

### ISSUE THREAD

**Issue by csadorf** [glotzerlab/signac#97]
**[Original report](https://bitbucket.org/glotzer/signac/issue/97) by Tim Moore (Bitbucket: [mootimot](https://bitbucket.org/mootimot), GitHub: [tcmoore3](https://github.com/tcmoore3)).**

----------------------------------------

I often find myself searching for jobs, and wanting to see particular state point variables of the jobs that are found. I currently pipe the output of `signac find` to `signac statepoint` and use bash commands to print what I want to see. For example:

```
$ signac find var1 val1 var2 val2 | xargs -I % bash -c "echo % && signac statepoint % -p | grep -e \'var3\' -e \'var4\'"
Interpreted filter arguments as '{"var1": val1, "var2": "val2"}'.
693669c869a969af698969d169636937
 'var3': 1.0,
 'var4': 2.0,
4ee6c67d930e007d9da858790061ec69
 'var3': 1.4,
 'var4': 2.0,
7959ea1762c8f3437647a94ce266b07a
 'var3': 1.0,
 'var4': 2.2,
8dbd8d710dc58a6969124983a3150014
 'var3': 1.0,
 'var4': 2.4,
```

It would be nice if there was a cleaner way to do this, e.g., something like
`$ signac find var1 val1 var2 val2 --show='var3 var4'`(but something different since `--show` is already an option to `find`).

**Comment by csadorf** [comment]
@vyasr Is this resolved?

**Comment by vyasr** [comment]
I had something, I'll cleanup and push and make a PR.

**Comment by csadorf** [comment]
Resolved with merge of PR #146 .
